src/pages/students.js

```diff
@@ -0,0 +1,346 @@
+              </p>
+            </div>
+            <Video id="irf10ez4h1" type="wistia" />
+          </section>
+          <section className={cx(styles.section, styles.stepsSection)}>
+            <h2>Access the New Relic Student Edition</h2>
+            <div className={styles.stepsListing}>
+              <div className={styles.signupSteps}>
+                <Surface
+                  base={Surface.BASE.SECONDARY}
+                  className={styles.signupStep}
+                >
+                  <div className={styles.stepNumber}>1</div>
+                  <h3 className={styles.stepHeader}>
+                    Signup for{' '}
+                    <em>
+                      <strong>FREE</strong>
+                    </em>{' '}
+                    Account
+                  </h3>
+                  <p className={styles.stepDetail}>
+                    To access the Student Edition you will need a New Relic
+                    free-tier account. After you will need to verify your
+                    student status.
+                  </p>
+
+                  <Button
+                    to="/"
+                    as={Link}
+                    variant={Button.VARIANT.PRIMARY}
+                    className={styles.stepButton}
+                  >
+                    Signup for a free account
+                  </Button>
+                </Surface>
+                <Surface
+                  base={Surface.BASE.SECONDARY}
+                  className={styles.signupStep}
+                >
+                  <div className={styles.stepNumber}>2</div>
+                  <h3 className={styles.stepHeader}>
+                    Verify Your Student Status
+                  </h3>
+                  <p className={styles.stepDetail}>
+                    The Student Edition is offered through the{' '}
+                    <a href="https://education.GitHub.com/benefits">
+                      GitHub Student Developer Pack
+                    </a>
+                    . If you don't have access to GitHub Students, and verify
+                    your account.
+                  </p>
+
+                  <Button
+                    to="/"
+                    as={Link}
+                    variant={Button.VARIANT.SECONDARY}
+                    className={styles.stepButton}
+                  >
+                    GitHub Student Developer Pack
+                  </Button>
+                </Surface>
+                <Surface
+                  base={Surface.BASE.SECONDARY}
+                  className={styles.signupStep}
+                >
+                  <div className={styles.stepNumber}>3</div>
+                  <h3 className={styles.stepHeader}>Enjoy Using New Relic</h3>
+                  <p className={styles.stepDetail}>
+                    After your status is verified. Your Student Edition account
+                    will be created giving you 3 full users and 500GB of data
+                    ingest.
```

Comment:
A few tips:
- Try to stick to present tense
- The first "sentence" is a fragment

For clarify, is this really creating a new account? Or is it more of granting certain privileges? I recommend updating to something like:

> After your status is verified, your account is upgraded to the Student Edition, giving you three full users and 500GB of data ingest.